tests/test_main.py

```diff
@@ -47,6 +47,11 @@ def test_global_arrival(self):
         self.assertTrue(lambda: callable(settings.PRISTINE_FUNCTION_SETTING))
         self.assertEqual(settings.PRE_SETUP_TEST_SETTING, 6)
 
+    def test_imagekit_app_config_working(self):
+        """Test whether app-config is correctly injecting the imagekit settings."""
+        from django.conf import settings
+        self.assertEqual(settings.IMAGEKIT_CACHE_BACKEND, 'default')
```

Comment:
Unfortunately (if you want so), this test passes. :worried:

Provided you have the `imagekit` app listed in your `INSTALLED_APPS` (see other comment above).